server/index.js

```diff
@@ -1,27 +1,34 @@
-'use strict';
+import loopback from 'loopback';
+import boot from 'loopback-boot';
 
-var loopback = require('loopback');
-var boot = require('loopback-boot');
-var debug = require('debug')('server');
+var app = loopback();
 
-var server = loopback();
-
-server.start = () => {
+app.start = function() {
```

Comment:
I reran the yeoman generator and didn't really go through this file but mostly what I did when I say 'converted to es6' is that I moved to the es6 modules instead of commonJS